plugins/BEdita/Core/src/Model/Table/ProfilesTable.php

```diff
@@ -38,6 +38,8 @@ public function initialize(array $config)
         $this->setPrimaryKey('id');
         $this->setDisplayField('name');
 
+        $this->addBehavior('BEdita/Core.Relations');
```

Comment:
I think that It should be added in every table that represents an object, i.e. in ObjectsTable, ProfilesTable, UsersTable, ...